MSAL/src/requests/MSALSilentRequest.m

```diff
@@ -209,10 +343,11 @@ - (void)acquireTokenWithRefreshToken:(MSIDRefreshToken *)refreshToken
 
 - (MSIDTokenRequest *)tokenRequest
 {
+    MSIDRefreshToken *refreshToken = [self.familyId isEqualToString:@"1"] ? self.familyRefreshToken : self.refreshToken;
```

Comment:
do we even need to have both familyRefreshToken and refreshToken properties? They're both refresh tokens...

Reply:
It helps in 2 ways. One in comparing the contents of MRT and FRT before and skip the network call if they are the same and second it helps in improving performance by using the already set value and not doing an additional cache look up.